### ISSUE TITLE
nerdfont-fontconfig missing icons

### ISSUE THREAD

**Issue by nate0001** [bhilburn/powerlevel9k#448]
I have one repo that was cloned from github not showing any vcs identifier. However, a repo with no remote url is showing the github icon.

![](http://i.imgur.com/QZe0l0u.png)

**Comment by dritter** [comment]
What is the `POWERLEVEL9K_MODE` you use? 
In `awesome-patched` all git repos show the octocat. Unfortunately the patched version do not contain the other glyphs. That could be the first part of the mystery..

Second part is the icon not being displayed when in repo with github remote. I just did a quick
```zsh
[[ "https://github.com/nate0001/dotfiles.git" =~ "github" ]] && echo true || echo false
```
which return `true` for me, so the comparison matched. So it should basically work. Are you sure you did not overwrite the default `POWERLEVEL9K_VCS_GIT_ICON`?

**Comment by nate0001** [comment]
Here's my settings. I've tried copying all the VCS icons from the utility file and they all displayed correctly.

```POWERLEVEL9K_MODE='nerdfont-fontconfig'
POWERLEVEL9K_LEFT_PROMPT_ELEMENTS=(time dir)
POWERLEVEL9K_RIGHT_PROMPT_ELEMENTS=(status vcs)
POWERLEVEL9K_STATUS_VERBOSE=false
POWERLEVEL9K_SHORTEN_DIR_LENGTH=3

POWERLEVEL9K_TIME_BACKGROUND="97"
POWERLEVEL9K_TIME_FOREGROUND="250"

POWERLEVEL9K_DIR_HOME_BACKGROUND="237"
POWERLEVEL9K_DIR_HOME_FOREGROUND="249"
POWERLEVEL9K_DIR_HOME_SUBFOLDER_BACKGROUND="237"
POWERLEVEL9K_DIR_HOME_SUBFOLDER_FOREGROUND="249"
POWERLEVEL9K_DIR_DEFAULT_BACKGROUND="237"
POWERLEVEL9K_DIR_DEFAULT_FOREGROUND="249"

POWERLEVEL9K_VCS_CLEAN_BACKGROUND='239'
POWERLEVEL9K_VCS_CLEAN_FOREGROUND='170'
POWERLEVEL9K_VCS_UNTRACKED_BACKGROUND='214'
POWERLEVEL9K_VCS_UNTRACKED_FOREGROUND='236'
POWERLEVEL9K_VCS_MODIFIED_BACKGROUND='196'
POWERLEVEL9K_VCS_MODIFIED_FOREGROUND='236'```

**Comment by nate0001** [comment]
No wonder. The nerdfont config only has VCS_GIT_ICON declared.

**Comment by dritter** [comment]
👍  Good catch! Even #439 does not add them.

**Comment by bhilburn** [comment]
Fixed by merging #451!

Thanks for reporting, @nate0001!
